sys/hacl/c/src/msvc/Hacl_Curve25519_64.c

```diff
@@ -27,9 +27,7 @@
 
 #include "internal/Vale.h"
 #include "internal/Hacl_Krmllib.h"
```

Comment:
Is this a manual change?

Reply:
There's no need to touch these files.
This makes reviewing things unnecessarily hard.
I'll update all the C files when this PR is in to make sure we have the correct code.
`config.h` is used in hacl packages and everywhere else.

Reply:
#67